XamlControlsGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -37,7 +34,21 @@
             <local:ControlExample.Substitutions>
                 <local:ControlExampleSubstitution Key="IsActive" Value="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" />
             </local:ControlExample.Substitutions>
-
+        </local:ControlExample>
+        <local:ControlExample HeaderText="A determinate progress ring.">
```

Comment:
I recommend running the Accessibility Insights app on your changes before submitting a PR. It's showing that both the ProgressRing and the NumberBox are missing automation names.

Reply:
Is there a way to fix that? I followed the sample from ProgressBar.

Reply:
For every control, you can always set AutomationProperties.Name="something useful" (where the name, ideally, is not just the control name repeated)